lua/orgmode/capture/init.lua

```diff
@@ -292,8 +307,11 @@ function Capture.autocomplete_refile(arg_lead)
 end
 
 function Capture:kill()
-  vim.cmd([[autocmd! OrgCapture BufWipeout <buffer>]])
-  vim.api.nvim_win_close(0, true)
+  vim.api.nvim_create_autocmd('BufWipeout', {
+    buffer = 0,
+    group = 'OrgCapture',
+    callback = 'nvim_win_close(0, true)',
```

Comment:
Are you sure this will work? I think `callback` needs to be lua function:
```lua
callback = function()
  return vim.api.nvim_win_close(0, true)
end
```

Reply:
I read in help that it does. but yeah, safer from lua. will fix